sockeye/data_io.py

```diff
@@ -1042,7 +1042,7 @@ def ids2tokens(token_ids: Iterable[int],
 """
 
     tokens = [vocab_inv[token] for token in token_ids]
-    return (tok for token_id, tok in zip(token_ids, tokens) if token_id not in exclude_set)
+    return [tok for token_id, tok in zip(token_ids, tokens) if token_id not in exclude_set]
```

Comment:
please leave this as a generator as discussed in the scoring PR

Reply:
Sorry about that - missed the discussion.